doc/other_modules/forge2d.md

```diff
@@ -65,57 +65,48 @@ not according to the coordinate system of Forge2D (where the Y-axis is flipped).
 
 ## Contact callbacks
 
-If you are using `Forge2DGame` you can take advantage of its way of handling contacts between two
-`BodyComponent`s.
+`Forge2DGame` provides a simple out of the box solution to propagate
+contact events.
```

Comment:
A bit more explanation of what a contact event is would be good

Reply:
The second comment here wasn't resolved I think?

Reply:
@spydon true that was because I messaged you off-github, about future plans.

Reply:
I think just a sentence would be good to have here (right after the first sentence) explaining it? And then it can just be moved together with all the docs later. Since there will be people using these docs before the other docs are ready.